commit/src/domain.rs

```diff
@@ -192,14 +192,26 @@ impl<Val: TwoAdicField> PolynomialSpace for TwoAdicMultiplicativeCoset<Val> {
     ) -> Vec<RowMajorMatrix<Self::Val>> {
         debug_assert_eq!(evals.height(), self.size());
         debug_assert!(log2_strict_usize(num_chunks) <= self.log_size());
-        // todo less copy
-        (0..num_chunks)
-            .map(|i| {
-                evals
-                    .as_view()
-                    .vertically_strided(num_chunks, i)
-                    .to_row_major_matrix()
-            })
+        let height = evals.height();
```

Comment:
Might be better? Not completely sure. These two should be equal but `evals.height()` is implemented via a len call and a division (which may not be by a power of `2`) whereas `self.size()` is just a shift so should probably be faster.
```suggestion
        let height = self.size();
```